### PR TITLE
[CLOUDTRUST-2944] Remove email/phoneNumber from KYC

### PR DESCRIPTION
**Pull request by fperot74**

### CODE REVIEW

**Review by fratt-elca — APPROVED** [review]
Retest ok

File: api/kyc/api.go [hunk 1]
```diff
@@ -199,8 +199,8 @@ func (u *UserRepresentation) Validate() error {
 		ValidateParameterRegExp(prmUserGender, u.Gender, regExpGender, true).
 		ValidateParameterRegExp(prmUserFirstName, u.FirstName, regExpFirstName, true).
 		ValidateParameterRegExp(prmUserLastName, u.LastName, regExpLastName, true).
-		ValidateParameterRegExp(prmUserEmail, u.Email, regExpEmail, true).
-		ValidateParameterPhoneNumber(prmUserPhoneNumber, u.PhoneNumber, true).
+		//ValidateParameterRegExp(prmUserEmail, u.Email, regExpEmail, false).
```
Comment (harture) [inline api/kyc/api.go:202]: forgot to remove it ?

File: api/management/api.go [hunk 2]
```diff
@@ -225,10 +225,11 @@ type BackOfficeConfiguration map[string]map[string][]string
 // RealmAdminConfiguration struct
 type RealmAdminConfiguration struct {
 	Mode                *string                   `json:"mode"`
-	AvailableChecks     map[string]bool           `json:"available-checks"`
+	AvailableChecks     map[string]bool           `json:"available_checks"`
```
Comment (harture) [inline api/management/api.go:228]: If wew want to change this, we need to synchronize with abilis as they are now using this attribute

Comment (harture) [reply api/management/api.go:228]: And sync with fratt for the BO

**Review by harture — APPROVED** [review]

**Pull request merged**